cafebabel/tests/utils.py

```diff
@@ -1,8 +1,10 @@
 def login(client, email, password):
-    return client.post('/login', data=dict(
+    response = client.post('/login', data=dict(
         email=email,
         password=password
     ), follow_redirects=True)
+    assert b'<form action="/login"' not in response.data, response.data
```

Comment:
Why not `assert 'action="/login"' not in response` ?

Reply:
We are out of the context where it works.